integration_test/cases/browser/refute_text_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Wallaby.Integration.Browser.RefuteTextTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  alias Wallaby.ExpectationNotMetError
+
+  test "refute_text/2 waits for presence of text and returns true if not found", %{session: session} do
+    element = session
+              |> visit("wait.html")
+              |> find(Query.css("#container"))
+
+    assert refute_text(element, "not-present")
```

Comment:
I'm not sure this test is really testing that it blocks properly. In order to validate this we'd need to start with some text on the screen and then ensure that refute_text blocks until that text is removed. As it is currently I don't think this will block correctly.